docs/plugins-create.md

```diff
@@ -88,6 +88,32 @@ Note: This will only apply for users who have enabled the `legacy_version_file`
 
 This can be used to further parse the legacy file found by asdf. If `parse-legacy-file` isn't implemented, asdf will simply cat the file to determine the version. The script will be passed the file path as its first argument.
 
+#### bin/plugin-add
+
+This can be used to run any post-installation actions after the plugin has been added to asdf.
+
+The script has access to the path the plugin was installed (`${ASDF_PLUGIN_PATH}`) and the source URL (`${ASDF_PLUGIN_SOURCE_URL}`), if any was used.
+
+See also the related hooks:
+
+* `pre_asdf_plugin_add`
+* `pre_asdf_plugin_add_${plugin_name}`
+* `post_asdf_plugin_add`
+* `post_asdf_plugin_add_${plugin_name}`
```

Comment:
@joschi @vic where are these documented? Seems odd to reference these hooks in the guide here if they aren't documented anywhere else.

Also, I think we should have pre and post in the names of the files:

- post-plugin-add
- pre-plugin-remove

Reply:
@jthegedus These aren't filenames but hook names which can be used in `.asdfrc`.

I didn't find any documentation for the other hooks (such as `pre_asdf_reshim_${plugin_name}` or `pre_asdf_uninstall_${plugin_name}` except for the changelog entries, so I decided not to spend too much time on documenting these either.

See also: https://github.com/asdf-vm/asdf/search?q=asdf_run_hook&type=Code

Reply:
> Also, I think we should have pre and post in the names of the files:
> 
> * post-plugin-add
> * pre-plugin-remove
>
> As it will better communicate the intent.

Yes, I think this makes a lot of sense. Should I create a follow-up PR?

Reply:
Another PR would be appreciated 🙏

Reply:
I will add documenting those missing hooks to my list of docs updates

Reply:
See #684.